CONTRIBUTING.rst

```diff
@@ -54,7 +54,7 @@ Install Werkzeug in editable mode::
 
 Install the minimal test requirements::
 
-    pip install pytest pytest-xprocess requests
+    pip install pytest pytest-xprocess requests six
```

Comment:
We do not use six. Add the specific thing you need to `werkzeug._compat`.

Reply:
Can do, it's only working in Python 2.7 anyway from what I've seen. Will have a look at this tomorrow.